docs/agents/ROCKETRIDE_PIPELINES.md

```
@@ -0,0 +1,1175 @@
+not override `model`/`modelTotalTokens` except with the `custom` profile:
+
+| Profile | Model | Context (total/output tokens) |
+| --- | --- | --- |
+| `openai-5-4`, `openai-5-4-pro` | gpt-5.4 / gpt-5.4-pro | 1,050,000 / 128,000 |
+| `openai-5-4-mini`, `openai-5-4-nano` | gpt-5.4-mini / -nano | 400,000 / 128,000 |
+| `openai-5-2` | gpt-5.2 (default profile) | 400,000 / 128,000 |
+| `openai-5-1` | gpt-5.1 | 400,000 / 128,000 |
+| `openai-5` | gpt-5 | 400,000 / 128,000 |
+| `openai-5-mini`, `openai-5-nano` | gpt-5-mini / gpt-5-nano | 400,000 / 128,000 |
+| `openai-4o`, `openai-4o-mini` | gpt-4o / gpt-4o-mini | 128,000 / 16,384 |
+| `custom` | you supply `model`, `modelTotalTokens`, `apikey` (see also `llm_openai_api` for OpenAI-compatible endpoints) |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 3; Actual: 2; Too few cells, row will be missing data

(MD056, table-column-count)